DayrSuryan/DSEthiop6.xml

```diff
@@ -304,7 +304,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                <history>
                   <origin>
-                     <origDate notBefore="1880" notAfter="1948" evidence="internal-date">Late 19th century to 1948.</origDate>
```

Comment:
the internal date evidence is based on the date 1948 in a1. If we change to a palaeographic dating, I would change it to lettering

Reply:
Done

Reply:
Now it is again 1948? I am confused...